scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -75,6 +89,36 @@ label {
   }
 }
 
+// Style new group of type radio
+.form-inline {
+
+  &.form-inline-tabs .form-group {
```

Comment:
I did this to avoid having 4 levels which is not recommended in SASS

```
.form-inline {
  &.form-inline-tabs {
    .form-group {
      box-shadow: $crm-form-control-shadow;
      border: 1px solid $crm-darker-grayblue;
      line-height: 1em;
      padding: 8px 15px;
      margin-right: -5px;
      min-width: 180px;

      &.active {
        background-color: $gray-lighter;
      }

      &:first-child {
        border-radius: 2px 0 0 2px;
      }

      &:last-child {
        border-radius: 0 2px 2px 0;
      }

      input {
        margin-right: 10px;
      }
    }
  }
}
```